### DISCUSSION TITLE
Make ktlint more opinionated

### DISCUSSION THREAD

**Discussion by hiqua** [iSoron/uhabits#714] [Feature Requests] [answered]
It is still possible to have discussions about the proper format of kotlin code, in spite of ktlint. There has to be a way to make it more opinionated, so that passing the CI means the format is exactly how it should be. (not sure whether this belongs here or as an issue).

I've noticed the following:
* if it fits as one line, should be one line
* only one statement per line (i.e. not `f(); g()`)

**Comment by iSoron** [comment]
@hiqua I agree. Pull requests are welcome for making ktlint more opinionated.

**Comment by hiqua** [comment, marked as answer]
https://github.com/pinterest/ktlint/issues/1078
https://github.com/pinterest/ktlint/issues/1077

For manually written code the current setup and improvements on out-of-the-box ktlint should be enough, it was mostly a problem with automatically generated code.
